src/decoration.c

```diff
@@ -78,10 +105,10 @@ EWRAM_DATA static u16 sDecorationsScrollOffset = 0;
 EWRAM_DATA u8 gCurDecorationIndex = 0;
 EWRAM_DATA static u8 sCurDecorationCategory = DECORCAT_DESK;
 EWRAM_DATA static u32 filler_0203a174[2] = {};
-EWRAM_DATA struct DecorationPCContext gDecorationContext = {};
+EWRAM_DATA static struct DecorationPCContext sDecorationContext = {};
 EWRAM_DATA static u8 sDecorMenuWindowIds[4] = {};
```

Comment:
I should introduce a `WINDOW_COUNT` to the enum, and use that in place of the 4 here.